src/styles/blocks/header.scss

```diff
@@ -0,0 +1,61 @@
+.container {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.logo_link {
+  height: 40px;
+}
+
+.nav {
+  margin: 0;
+
+  &__links {
+    display: flex;
+    margin: 0;
+    padding: 0;
+    list-style-type: none;
+
+    & li:last-child a {
+      margin-right: 0;
+    }
```

Comment:
The selector '& li:last-child a' is incorrect. In SCSS, '&' refers to the parent selector, so your code will compile to '.nav__links li:last-child a'. According to BEM methodology, you should avoid creating selectors that increase specificity. It's better to create a new class for the last child link and style it accordingly.